web/src/lib/components/assets/thumbnail/thumbnail.svelte

```diff
@@ -252,8 +256,7 @@
         {#if !usingMobileDevice && mouseOver && !disableLinkMouseOver}
           <!-- lazy show the url on mouse over-->
           <a
-            class="absolute w-full top-0 bottom-0"
-            style:cursor="unset"
+            class="absolute w-full top-0 bottom-0 z-10"
```

Comment:
Uhm?

Reply:
Opps. Meaning to change the zindex only

Reply:
and you need at least z-1 here so that the URL information can be seen at the page's lower left corner